### ISSUE TITLE
Endless re-render axis Bottom date

### ISSUE THREAD

**Issue by DrWeyber1903** [plouc/nivo#1478]
Hello everybody
Faced the problem of constantly redrawing time data. Found no dependencies.
Strong freezes appear and it is impossible to interact with the page

![image](https://user-images.githubusercontent.com/74793279/114520076-b77e8580-9c49-11eb-997d-f2d33b291b86.png)

 ```
 <ResponsiveLine
      data={data}
      margin={{
        top: 20,
        right: 20,
        bottom: 70,
        left: 45,
      }}
      yScale={{
        type: 'linear',
        min: 'auto',
        max: 'auto',
        reverse: false,
        stacked: true,
      }}
      curve="monotoneX"
      axisTop={null}
      axisRight={null}
      xScale={{
        type: 'time',
        format: '%Y-%m-%d %H:%M:%S',
        precision: 'minute',
      }}
      xFormat="time:%Y-%m-%d %H:%M:%S"
      axisBottom={{
        orient: 'bottom',
        tickSize: 0,
        tickPadding: 10,
        tickRotation: -90,
        tickValues = 'every 5 minute',
        format = '%H %M,
        legendOffset: 100,
        translateX: 25,
        legendPosition: 'start',
      }}
      axisLeft={{
        orient: 'left',
        tickSize: 5,
        tickPadding: 5,
        tickRotation: 0,
        legend: '',
        legendOffset: -40,
        legendPosition: 'middle',
      }}
      enableGridY={false}
      enableGridX={false}
      lineWidth={2}
      enablePoints={false}
      pointSize={5}
      pointBorderWidth={2}
      pointLabelYOffset={-12}
      isInteractive={false}
      useMesh
      enableCrosshair={false}
      legends={[]}
      animate={false}
    />
```

Render data
![image](https://user-images.githubusercontent.com/74793279/114520678-5b683100-9c4a-11eb-9b99-ba71755cf942.png)

**Comment by DrWeyber1903** [comment]
Downgrade to 0.63.1, and loop is gonna out

**Comment by inrumi** [comment]
@DrWeyber1903 I think this was fixed here #1379

**Comment by plouc** [comment]
2 fixes have been merged regarding date issues leading to endless re-rendering, there should be a release containing those fixes soon.
